### ISSUE TITLE
HassOS boot on sd card but not on emmc on odroid xu4

### ISSUE THREAD

**Issue by MattL0** [home-assistant/operating-system#558]
is is working on emmc? Nothing is booting.

**Comment by MattL0** [comment]
I tried different version on emmc nothing is booting

**Comment by pvizeli** [comment]
XU4 is very complex for booting with eMMC unlike the C2/N2 images they work on SD like on eMMC. See at: https://github.com/home-assistant/operating-system/blob/dev/Documentation/boards/hardkernel/odroid-xu4.md

**Comment by Joentje** [comment]
I'm having the same issues. Flashed HassOS 3.8 and 4.0. Same result. After flashing 4.0 it was complaining the checksum was not matching in Etcher.

I followed all the steps from: https://github.com/home-assistant/operating-system/blob/dev/Documentation/boards/hardkernel/odroid-xu4.md

When I flashed HassOS on a SD card and the emmc, and booted from SD. The `/dev/mmcblk0` did not exists. So the I used the `dd if=/dev/mmcblk1 of=/dev/mmcblk0boot0 bs=512 skip=1 seek=0 count=16381` command. That didn't work either.

After flashing Ubuntu, it is just working fine.

So no I'm a bit lost :)
